inspectors/view/qml/notation/mmrests/MMRestPopup.qml

```diff
@@ -0,0 +1,83 @@
+import QtQuick 2.9
+import QtQuick.Controls 2.2
+import MuseScore.Inspectors 3.3
+import MuseScore.UiComponents 1.0
+import MuseScore.Ui 1.0
+import "../../common"
+
+StyledPopup {
+    id: root
+
+    property QtObject model: null
+
+    implicitHeight: contentColumn.implicitHeight + topPadding + bottomPadding
+    width: parent.width
+
+    Column {
+        id: contentColumn
+
+        width: parent.width
+
+        spacing: 12
+
+        Column {
+            spacing: 8
+
+            height: childrenRect.height
+            width: parent.width
+
+            Item {
+                height: childrenRect.height
+                width: parent.width
+
+                Column {
+                    anchors.left: parent.left
+                    anchors.right: parent.horizontalCenter
+                    anchors.rightMargin: 2
+
+                    spacing: 8
+
+                    CheckBox {
+                        id: numberVisibilityCheckBox
+
+                        isIndeterminate: root.model ? root.model.isNumberVisible.isUndefined : false
+                        checked: root.model && !isIndeterminate ? root.model.isNumberVisible.value : false
+                        text: qsTr("Number visible")
+
+                        onClicked: { root.model.isNumberVisible.value = !checked }
+                    }
+                }
+
+                Column {
+                    anchors.left: parent.horizontalCenter
+                    anchors.leftMargin: 2
+                    anchors.right: parent.right
+
+                    spacing: 8
+
+                    StyledTextLabel {
+                        text: qsTr("Number position")
+                    }
+
+
+                    IncrementalPropertyControl {
+                        id: numberPositionControl
+
+                        icon: IconCode.VERTICAL
+                        enabled: root.model ? model.isEmpty || numberVisibilityCheckBox.checked : true
+                        isIndeterminate: root.model ? root.model.numberPosition.isUndefined : false
+                        currentValue: root.model ? root.model.numberPosition.value : 0
+
+                        measureUnitsSymbol: "sp"
```

Comment:
As far as I remember, @Tantacrul was completely against using 'sp' in the inspector, because it makes no sense to 90% of users.

Reply:
I'll remove it, then.